frontend/src/components/user/ATSUserDetailsModal.vue

```diff
@@ -104,27 +84,27 @@ watch(visible, () => {
       </template>
 
       <Column
-        field="atsClientNumber"
+        field="clientId"
         header="Client #"
       />
       <Column
         field="firstName"
         header="First Name"
       />
       <Column
-        field="lastName"
+        field="surName"
         header="Last Name"
       />
       <Column
         field="phone"
         header="Phone"
       />
       <Column
-        field="email"
+        field="address.email"
```

Comment:
`email` exists in both `ATSClientResource` and `AddressResource` for whatever reason. Just want to confirm this is the correct one to use?

Reply:
yes, address.email is the correct one to use for this modal as it is fetched from ATS api.